src/lightning/pytorch/callbacks/early_stopping.py

```diff
@@ -123,7 +122,7 @@ def __init__(
             raise MisconfigurationException(f"`mode` can be {', '.join(self.mode_dict.keys())}, got {self.mode}")
 
         self.min_delta *= 1 if self.monitor_op == torch.gt else -1
-        torch_inf = torch.tensor(np.Inf)
+        torch_inf = torch.tensor(float("inf"))
```

Comment:
You could also do:
```suggestion
        torch_inf = torch.tensor(torch.inf)
```
which perhaps is a tiny bit cleaner to read (same for nan).

Reply:
I have replaced the use of float with torch